rtcstats.js

```diff
@@ -212,6 +212,29 @@ module.exports = function(wsURL, getStatsInterval, prefixesToWrap) {
         }
       });
 
+      ['addTrack'].forEach(function(method) {
+        if (origPeerConnection.prototype[method]) {
+          var nativeMethod = pc[method];
+          pc[method] = function() {
+            var track = arguments[0];
+            var streams = [].slice.call(arguments, 1);
+            trace(method, id, track.kind + ':' + track.id + ' ' + streams.map(function(s) { return 'stream:' + s.id; }));
```

Comment:
how is a list serialized to string in JS?   shouldn't you do a .join(' ') and force whatever separator you want?